AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -19,6 +19,9 @@ amp-reddit-com.cdn.ampproject.org##.AppSelectorModal__body
 reddit.com#%#(function() { try { if ("undefined" != typeof localStorage) { localStorage.setItem("bannerLastClosed", "Thu Sep 16 2030 16:46:52 GMT+0300") } } catch (ex) {} })();
 reddit.com#%#(function() { try { if ("undefined" != typeof localStorage) { localStorage.setItem("pillLastClosed", "Thu Sep 16 2030 16:46:52 GMT+0300") } } catch (ex) {} })();
 !
+m.webtoons.com##.pop_download_home
+m.webtoons.com##._appDownloadPopup
+m.webtoons.com#$#.dim.home { display: none !important; }
```

Comment:
1. Please move this rule to those rules.
```
m.webtoons.com#$#._homeAppDownloadPopup { display: none!important; }
m.webtoons.com#$#body { overflow: visible!important; position: relative!important; }
```
2. Add comment with task link.
3. Change a bit the rule to the
```
m.webtoons.com#$#body { overflow: visible !important; position: relative !important; }
```